### DISCUSSION TITLE
Data from other market

### DISCUSSION THREAD

**Discussion by longlie1109** [alvarobartt/investiny#33] [Q&A]
Is there a way to get the "exchange" in search_assets function ?
NASDAQ works fine but other market isn't work out for me

**Comment by alvarobartt** [comment]
Hi, @longlie1109 can you please let me know which exchanges are not working fine for you? Thanks :smile:

**Reply by alvarobartt** [reply]
So I've seen that there's an open issue at #34, which I assume is directly related to what you're asking in this discussion, so I'll create another issue out of this discussion :smile:
